### ISSUE TITLE
Could not find base class "CustomRunner"

### ISSUE THREAD

**Issue by GammaHaya** [KoBeWi/Metroidvania-System#37]
Godot 4.2.1.

I have this issues in different projects with this plugin (and in some not).
When I start Godot I got these error messages:

```
  res://CustomRunnerIntegration/Config.gd:1 - Parse Error: Could not find base class "CustomRunner".
  modules/gdscript/gdscript.cpp:2788 - Failed to load script "res://CustomRunnerIntegration/Config.gd" with error "Parse error". (User)
```
Is there a known fix for this?

**Comment by KoBeWi** [comment]
You can safely delete the script if you are not using CustomRunner addon. Or comment out the first line in the script.
I'll come up with some proper fix.

**Comment by KoBeWi** [comment]
Should be fixed by aea65de
Make sure the `Config.gd` is not opened in the editor to avoid errors.
